internal/common/disasters/commonpooldefinition.go

```diff
@@ -0,0 +1,55 @@
+package disasters
+
+import (
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+type Commonpool struct {
```

Comment:
```suggestion
type CommonPool struct {
```
would be better :)

Reply:
sorry for the uglyness of our code guys!

Reply:
no worries, moof fas brek ting 
![image](https://user-images.githubusercontent.com/33488131/102802634-dca36b80-43ae-11eb-9145-e5bf09a42f7f.png)